syntax/rust.vim

```diff
@@ -43,8 +44,6 @@ syn keyword   rustKeyword     mod trait nextgroup=rustIdentifier skipwhite skipe
 syn keyword   rustStorage     move mut ref static const
 syn match rustDefault /\<default\ze\_s\+\(impl\|fn\|type\|const\)\>/
 
-syn keyword   rustInvalidBareKeyword crate
-
```

Comment:
Remember to remove its `hi def link` line too.

Reply:
Should be fixed now.